### ISSUE TITLE
[BUG] Title of the bug

### ISSUE THREAD

**Issue by taluripooja** [conda/grayskull#473]
I encountered an issue while trying to use the online version of the Grayskull, specifically regarding the generation of a recipe for the Flask-Minify package.

Upon attempting to generate the recipe, I consistently received a "Failed to Fetch" error message. I have made several attempts, but unfortunately, the problem persists. 

 If there are any workarounds or alternative methods to generate a recipe for Flask-Minify, I would be grateful for your suggestions.

**Comment by marcelotrevisani** [comment]
Yeah, the online version is down for a while, I have plans to redo it, so it might take a while until it is back again. In the meantime you can try to use grayskull in the command line :)
